mlflow/keras.py

```diff
@@ -13,6 +13,10 @@
 import importlib
 import os
 import yaml
+import gorilla
+import warnings
+from keras.callbacks import Callback
```

Comment:
Ah I remember why we don't want to import keras at the top-level here - it's so that `mlflow.keras.log_model` can be called on a `tf.keras` model with just TensorFlow installed (& not Keras). So we should probably put our import under `__MLflowKerasCallback`